### PR TITLE
Fix show fields

### LINKED ISSUE #58: Get vocabulary view and form showing all desired properties
But... do we want all properties shown?  Just title and publisher?  (That's what OD's JSONLD terms use)  Is the form going to be using the same properties as the view?

### PR DESCRIPTION
**Pull request by terrellt**
TermFactory will now spin up a Vocabulary object if appropriate.

Closes #58

### CODE REVIEW

**Comment by terrellt** [comment]
Hold off on this, I just found another use for PolymorphicTermRepository, needs some refactoring.

**Comment by terrellt** [comment]
Okay, this should be ready for review.

File: app/services/polymorphic_term_repository.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  end
+
+  def find
+    repository.find(id)
+  end
+
+  private
+
+  def repository
+    return vocabulary_repository unless id.include?("/")
```
Comment (jechols) [inline app/services/polymorphic_term_repository.rb]: I don't like this logic here, it feels sort of hard-coded for how important the switch is.  Can we make something that tells us when an id indicates a vocabulary vs. a term?  Isn't TermUri sort of on that path already?

Comment (jechols) [inline app/services/polymorphic_term_repository.rb]: Maybe not "hard-coded", but it seems like it's more important than just this one function

Comment (terrellt) [inline app/services/polymorphic_term_repository.rb]: This is a great point. TermUri is definitely close to this, but right now it works on the idea that it gets passed a full URI. Any recommendations? A similar value object?

Comment (jechols) [inline app/services/polymorphic_term_repository.rb]: Yeah, I thought about that... not sure the right approach - maybe a TermID class that TermUri and PolymorphicTermRepository can both use?

Comment (jechols) [inline app/services/polymorphic_term_repository.rb]: Oh, or maybe not.. TermUri doesn't really do quite the same kind of operation, it's all about the ending of the id....  Hmm.

**Pull request merged**